src/vl-form.js

```diff
@@ -0,0 +1,67 @@
+import {nativeVlElement, define} from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlForm
+ * @class
+ * @classdesc Formulier element.
+ *
+ * @extends HTMLElement
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-form/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-form/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-form.html|Demo}
+ *
+ */
+export class VlForm extends nativeVlElement(HTMLFormElement) {
+  static get _observedAttributes() {
+    return ['target', 'action'];
+  }
+
+  static get _targetElementId() {
```

Comment:
Misschien beetje verwarrend dat hier Id staat in functienaam, terwijl dit overal als name attribuut wordt gebruikt

Reply:
Het is de targetElementIdentifier, afgekort naar Id, maar ik zal het aanpassen naar targetElementName.